### PR TITLE
Patch 5

### PR DESCRIPTION
**Pull request by sandervandegeijn**
### Description
Undocumented feature to control bulk size.

### Issues Resolved
none

### Version
logstash output plugin

### Frontend features
N/A

### Checklist
- [ * ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks, @sandervandegeijn! I will get it reviewed and merged.

**Comment by hdhalter** [comment]
Hi @dlvenable, can you please review and approve this PR? Thanks!

File: _tools/logstash/ship-to-opensearch.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ output {
 - service_name (string, default => "es") - Defines the service name to be used for `aws_iam` authentication.
 - legacy_template (boolean, default => true) - Selects the OpenSearch template API. When `true`, uses legacy templates via the _template API. When `false`, uses composable templates via the _index_template API.
 - default_server_major_version (number) - The OpenSearch server major version to use when it's not available from the OpenSearch root URL. If not set, the plugin throws an exception when the version can't be fetched.
+- target_bulk_bytes (number) - number of maximum bytes in the buffer. After this has been reached it will flush the data. Useful when the bulk requests are too large for the opensearch cluster.
```
Comment (Naarcha-AWS) [inline _tools/logstash/ship-to-opensearch.md:153]:
```suggestion
- target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached Opensearch will flush the data. This is useful for when the bulk requests are too large for the OpenSearch cluster.
```

**Review by Naarcha-AWS — APPROVED** [review]

File: _tools/logstash/ship-to-opensearch.md [hunk 2]
```diff
@@ -150,6 +150,7 @@ output {
 - service_name (string, default => "es") - Defines the service name to be used for `aws_iam` authentication.
 - legacy_template (boolean, default => true) - Selects the OpenSearch template API. When `true`, uses legacy templates via the _template API. When `false`, uses composable templates via the _index_template API.
 - default_server_major_version (number) - The OpenSearch server major version to use when it's not available from the OpenSearch root URL. If not set, the plugin throws an exception when the version can't be fetched.
+- target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached Opensearch will flush the data. This is useful for when the bulk requests are too large for the OpenSearch cluster.
```
Comment (Naarcha-AWS) [inline _tools/logstash/ship-to-opensearch.md:153]:
```suggestion
- target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached OpenSearch will flush the data. This is useful for when the bulk requests are too large for the OpenSearch cluster.
```

**Comment by Naarcha-AWS** [comment]
@sandervandegeijn: You might have to open this in a new PR/branch. Looks like there was a rebase error.

**Comment by sandervandegeijn** [comment]
Shit, I tried to fix the DCO. Made a mistake. my bad.

**Pull request closed without merging**